src/store/annotation.ts

```diff
@@ -282,6 +282,41 @@ export class Annotations extends VuexModule {
     this.setSelected([]);
   }
 
+  @Action
+  public async tagAnnotationIds({annotationIds, tags}: {annotationIds: string[], tags: string[]}) {
+    sync.setSaving(true);
+    await Promise.all(
+      annotationIds
+        .map(annotationId => this.annotations.findIndex((annotation: IAnnotation) => annotation.id === annotationId))
```

Comment:
Don't you think it could be useful to have a helper method to get annotations from annotationsIds ?

Reply:
In this case I need the index to use it later but that's something that could be helpful in other places in the code
But I think it should be done in another pull request with eventually other reformatting